### PR TITLE
Single Post page, post comment feature and User Posts in profile page

### PR DESCRIPTION
**Pull request by kjais1720**
## Overview
- Created 
  - `createComment` component to post comments on a post
  - `ProfilePosts` component to render different types of posts on a user profile (posts, likes & bookmarks)
## Preview
image

## Preview link
- https://hooked-social-git-posts-kjais1720.vercel.app/home

### CODE REVIEW

**Review by sharathnair9999 — APPROVED** [review]
UI looks pretty cool. Good work

**Review by KanekarSnehal — APPROVED** [review]
UI looks great.
Code is good to merge.

File: src/components/CreateComment.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+    dispatch(postComment({ postId, formData }));
+    setFormData({content:""})
+  };
+  return (
+    <div>
+      <article
+        className="hidden rounded-2xl bg-light-100 p-2
```
Comment (ankit9015) [inline src/components/CreateComment.jsx:27]: I like that you are using tailwind

File: src/components/ImageSlider.jsx [hunk 2]
```diff
@@ -31,16 +31,15 @@ export function ImageSlider({ images }) {
       <div
         className={`ease flex gap-2 transition-all duration-300 ${x_position[activeImg]}`}
       >
-      
         {images.map(({ src, title }, idx) => (
           <figure
+            key={idx}
```
Comment (ankit9015) [inline src/components/ImageSlider.jsx:36]: I like that you are using idx attribute from the map method to add keys for the images

File: src/components/ProfilePosts.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  };
+  const postTypeToShow =
+    username === undefined
+      ? pathname.slice(9)
+      : pathname.slice(10 + username.length);
+  const postsToShow = postTypes[postTypeToShow];
+  const arePostsLoading =
+    (status.type === "getAllPosts" && status.value === "pending") ||
+    postsToShow === undefined;
+  const profileSectionButtons = [
```
Comment (ankit9015) [inline src/components/ProfilePosts.jsx:34]: This is an interesting approach for buttons

**Review by ankit9015 — APPROVED** [review]

**Comment by ankit9015** [comment]
In the preview, couldn't find the option to login as a guest

**Pull request merged**